mlir/include/mlir/Dialect/Tosa/IR/TosaComplianceData.h

```diff
@@ -0,0 +1,403 @@
+// The profile-based compliance content below is auto-generated by a script
+// in https://git.mlplatform.org/tosa/specification.git
```

Comment:
It might be worth pointing to the exact script so other users are able to use it if needed

Reply:
Added. Thanks.